testcases/request/body/matches with floats.json

```diff
@@ -0,0 +1,31 @@
+{
+  "match": true,
+  "comment": "Request matches with floats",
+  "expected": {
+    "headers": {"Content-Type": "application/json"},
+    "matchingRules": {
+      "$.body.alligator.feet": {"match": "regex", "regex": "\\d(\\.\\d{1,2})"}
```

Comment:
`$.body.alligator.feet` does not match the body. Should it be `$.body.product.price`?